src/ReactiveMarbles.Mvvm/ReactiveMarbles.Mvvm.csproj

```diff
@@ -1,7 +1,7 @@
 <Project Sdk="Microsoft.NET.Sdk">
 
     <PropertyGroup>
-      <TargetFrameworks>netstandard2.1;net5.0</TargetFrameworks>
+      <TargetFrameworks>netstandard2.0;netstandard2.1;net5.0</TargetFrameworks>
```

Comment:
Just use netstandard2.0;net5.0;net6.0

netstandard2.1 doesn't gain us anything

Reply:
Yeah.  I added 2.0 because I realized somehow it got set to 2.1.  I will go through and remove it from this and the ServiceLocator if it exists.